### ISSUE TITLE
Installation error messages with version 1.0.2

### ISSUE THREAD

**Issue by level420** [TLWebdesign/Joomla-3-EOL-Security-Fixes#4]
On my Joomla 3.10.12 instance I'm getting the following error messages when installing the plugin for all files:
```
JFile: :copy(/var/html/mysite.com[TMP]/install_65dc7da91e6d5/Joomla-3-EOL-Security-Fixes-1.0.2/files/administrator/components/com_banners/models/forms/banner.xml, [ROOT]//administrator/components/com_banners/models/forms/banner.xml): copy([ROOT]//var/html/mysite.com[TMP]/install_65dc7da91e6d5/Joomla-3-EOL-Security-Fixes-1.0.2/files/administrator/components/com_banners/models/forms/banner.xml): failed to open stream: No such file or directory
```
What could the cause be for this?

PS.: a big thanks for publishing this plugin!

**Comment by TLWebdesign** [comment]
Hi,

two questions: what is your environment, php / database / webserver / os? 
And can you send me an uncensored version of the error because it seems you left out more than i would expect. My e-mail address is in my profile. 

I just applied it again on my linux server and also MAMP PRO without any issues.

**Comment by trob** [comment]
I just tested:
It's installing totally fine on a Linux server (including showing the new version).

It's failing on my local Windows xampp.
In my case with error messages like "original File not found: C:\xampp8\htdocs\j3\tmp\install_65dc928224bd4\Joomla-3-EOL-Security-Fixes-main\files/administrator/components/com_banners/models/forms/banner.xml" on all files.
Says "Installation sucessfull..." but doesn't show the modified version.

**Comment by level420** [comment]
@TLWebdesign 

I'm having:
php 7.4 running in a docker image based on almalinux 8
mariadb 5.5
apache httpd 2.4.6

I've modified ```script.php``` in line https://github.com/TLWebdesign/Joomla-3-EOL-Security-Fixes/blob/main/script.php#L38 by ommitting the last 2 parameters of ```File::copy```, thus not using streams for copying and now it works as expected. 

What was reason in choosing copying to use streams?

**Comment by TLWebdesign** [comment]
The lack of knowledge knowing what streams meant to be honest! After reading up on this i believe it should be disabled and i'll do so in a newer version. Thanks for reporting this issue 👍

**Comment by TLWebdesign** [comment]
Just released V1.0.3.

**Comment by level420** [comment]
Works now  without problems! Thank you @TLWebdesign!
